packages/big-design/src/components/StatelessPagination/StatelessPagination.tsx

```diff
@@ -0,0 +1,102 @@
+import {
+  ArrowDropDownIcon,
+  ChevronLeftIcon,
+  ChevronRightIcon,
+} from '@bigcommerce/big-design-icons';
+import React, { memo } from 'react';
+
+import { MarginProps } from '../../helpers';
+import { Dropdown, DropdownItem } from '../Dropdown';
+import { Flex, FlexItem } from '../Flex';
+
+import { StyledButton } from './styled';
+
+export interface StatelessPaginationLocalization {
+  previousPage: string;
+  nextPage: string;
+}
+
+const defaultLocalization: StatelessPaginationLocalization = {
+  previousPage: 'Previous page',
+  nextPage: 'Next page',
+};
+
+export interface StatelessPaginationProps extends MarginProps {
+  itemsPerPage: number;
+  itemsPerPageOptions: number[];
+  onItemsPerPageChange(range: number): void;
+  disableNext?: boolean;
+  onNext(): void;
+  disablePrevious?: boolean;
+  onPrevious(): void;
+  label?: string;
+  rangeLabel?: string;
+  localization?: StatelessPaginationLocalization;
+}
+
+export const StatelessPagination: React.FC<StatelessPaginationProps> = memo(
+  ({
+    itemsPerPage,
+    itemsPerPageOptions,
+    disableNext = false,
+    onNext,
+    disablePrevious = false,
+    onPrevious,
+    onItemsPerPageChange,
+    label = 'pagination',
+    localization = defaultLocalization,
+    rangeLabel,
+  }) => {
+    const handleRangeChange = (item: DropdownItem) => {
+      onItemsPerPageChange(Number(item.hash));
+    };
+
+    return (
+      <Flex aria-label={label} flexDirection="row" role="navigation">
+        <FlexItem>
+          <Dropdown
+            items={itemsPerPageOptions.map((range) => ({
+              content: `${range}`,
+              hash: `${range}`,
+              onItemClick: handleRangeChange,
+            }))}
+            positionFixed={true}
+            selectedItem={{
+              content: `${itemsPerPage}`,
+              hash: `${itemsPerPage}`,
+              onItemClick: handleRangeChange,
+            }}
+            toggle={
+              <StyledButton
+                iconRight={<ArrowDropDownIcon size="xxLarge" />}
+                type="button"
+                variant="subtle"
+              >
+                {rangeLabel || `Show ${itemsPerPage} items`}
```

Comment:
Is it worth pulling out `Show ${itemsPerPage} items` into a `defaultLabel` variable to make it a tad more explicit? I'd imagine we'd set the `rangeLabel` prop to `rangeLabel = "Show ${itemsPerPage} items" ` if we could.

Reply:
turns out, we can: https://github.com/bigcommerce/big-design/pull/1598/commits/34b20eeaa9cdd0b63944fe41fd1e4b6ace5e8219

I've never thought to try using other variables within a defaulting before 🤔

Reply:
😮